DemoApp/src/SendText.cc

```diff
@@ -31,10 +31,13 @@ ViewPtr SendText::runAction() {
         puts(e.what());
     }
 
-    std::cout << setFormatting({ConsoleFormat::T_YELLOW})
-              << "\n\n Message sent successfuly, press enter to return to the previous view..." << clearFormatting();
+    if (!Utils::isCLIMode()) {
+        std::cout << setFormatting({ConsoleFormat::T_YELLOW})
+                  << "\n\n Message sent successfuly, press enter to return to the previous view..."
+                  << clearFormatting();
 
-    Utils::waitForEnter();
+        Utils::waitForEnter();
+    }
 
     return parent;
 }
```

Comment:
newline :P

Reply:
Not my changes, it was'n there even before this PR. Its your code btw